keps/prod-readiness/sig-apps/4017.yaml

```diff
@@ -1,3 +1,5 @@
 kep-number: 4017
 beta:
+  approver: "@wojtek-t"
+stable:
```

Comment:
Can't comment on unchanged lines, so adding the questions here:

> It will be tested manually prior to beta launch.

Has this happened?
Please update the response to describe the test together with the findings.

Reply:
Cc @danielvegamyhre

Reply:
@wojtek-t I tracked back the feature in beta to this PR https://github.com/kubernetes/kubernetes/pull/119232 but I do not find any comments on the manual testing done.

As a way to move forward, once the feature gate it locked, would it be okay to document the results as upgrade from beta -> GA of this feature?

Reply:
@wojtek-t how does one test upgrades with local environments?

Reply:
I think it's already a bit too late - if we push that as GA criteria, I'm afraid it will just not happen at all.

Regarding testing with local environments, there are many examples of that across different KEPs. A random example with kind is here:
https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/1669-proxy-terminating-endpoints#were-upgrade-and-rollback-tested-was-the-upgrade-downgrade-upgrade-path-tested

It shouldn't really take more than an hour or so to do some manual test and validate it.

Reply:
@wojtek-t from your provided example, I have updated the `###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?` section, PTAL